client.py

```diff
@@ -58,20 +74,21 @@ def get_version(repo_name, repo_team):
         sha = tags.commit.sha
         date = tags.commit.commit.last_modified
         author = tags.commit.author.login
+        latest_release = {}
         if iteration == 0:
-            latestrelease = {'version': version,
+            latest_release = {'version': version,
                              'sha': sha,
                              'date': date,
                              'author': author
                              }
             iteration = 1
         elif iteration == 1:
-            previousrelease = {'version': version,
+            previous_release = {'version': version,
                                'sha': sha,
                                'date': date,
                                'author': author
                                }
-            return {'LatestRelease': latestrelease, 'PreviousRelease': previousrelease}
+        return {'LatestRelease': latest_release, 'PreviousRelease': previous_release}
```

Comment:
this line was correctly placed before. It must be in the 'elif:' block, otherwise the function returns only the latest release